west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 3dd7043488c5d965284d0bc13926e633d2919f65
```

Comment:
I don't think we can bump the Zephyr revision without an upmerge.

Reply:
It seems to be possible: https://github.com/nrfconnect/sdk-nrf/commit/c99f248307f89d20a06a0aaf3b68e40d18712d06 https://github.com/nrfconnect/sdk-nrf/commit/b152f3d4d5e36917dd21428fa123717a1b524050

Reply:
This is ok. This PR depends on latest commmits in sdk-zephyr.